app/services/query-service.js

```diff
@@ -80,23 +80,52 @@ const runFetchAnswerQuery = async ({ query, chatHistory, summariesMode, embeddin
   } catch (error) {
     trackFetchResponseFailed({
       errorMessage: error.message,
-      requestQuery: query
+      requestQuery: query,
+      retryCount
     })
     return {
       response: { answer: 'This tool cannot answer that kind of question, ask something about Defra funding instead', hallucinated: true }
     }
   }
 }
 
-/**
- * Fetches answer from AI Search and OpenAI
- * @param {*} req
- * @param {string} query
- * @param {BaseMessage[]} chatHistory
- * @param {boolean} cacheEnabled
- * @param {boolean} summariesEnabled
- * @returns {{ response: string, summariesMode: boolean, hallucinated: boolean }}
- */
+const runFetchAnswer = async ({ query, chatHistory, cacheEnabled, summariesEnabled, embeddings, model, retryCount }) => {
+  try {
+    if (summariesEnabled) {
+      const { response: summariesResponse, hallucinated } = await runFetchAnswerQuery({ query, chatHistory, summariesMode: summariesEnabled, model, embeddings, retryCount })
+
+      if (!hallucinated) {
+        // TODO cache summaries response after enabled
+        return {
+          response: summariesResponse?.answer,
+          summariesMode: true,
+          hallucinated
+        }
+      }
+    }
+
+    summariesEnabled = false
```

Comment:
Instead of rewriting this variable, just run `runFetchAnswerQuery` with `summariesMode: false`?

Reply:
this is for the
```
catch (error) {
    return {
      response: JSON.stringify({ answer: 'This tool cannot answer that kind of question, ask something about Defra funding instead', items: [] }),
      hallucinated: true,
      summariesMode: summariesEnabled
    }
  }
```

to know if summaries was enabled when broken